app/build.gradle.kts

```diff
@@ -197,6 +197,12 @@ android {
     create("production") {
       dimension = "track"
     }
+
+    create("tamilNadu") {
+      dimension = "track"
+      applicationIdSuffix = ".tamilnadu"
+      versionNameSuffix = "-tamilNadu"
```

Comment:
We don't need to add version name suffix. We can follow the same pattern as production apps for version name.